app/application/usecase/account/get_balance_test.go

```diff
@@ -0,0 +1,64 @@
+package account
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"stoneBanking/app/domain/entities/account"
+	"stoneBanking/app/domain/entities/token"
+	"stoneBanking/app/domain/types"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_GetBalance(t *testing.T) {
+	testCases := []struct {
+		name        string
+		accountMock account.Repository
+		tokenMock   token.Repository
+		input       string
+		runBefore   func(db *sql.DB)
+		want        float64
+		wantErr     bool
+	}{
+		{
+			name: "dado id externo correto, retorna o valor da conta apropriadamente",
+			accountMock: &account.RepositoryMock{
+				GetByIDFunc: func(ctx context.Context, accountID types.ExternalID) (account.Account, error) {
+					account := account.Account{
+						ID:         1,
+						Name:       "Joao do Rio",
+						ExternalID: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJVc2VySUQiOiJjMDM2NDc1Zi1iN2EwLTRmMzQtOGYxZi1jNDM1MTVkMzE3MjQifQ.Vzl8gI6gYbDMTDPhq878f_Wq_b8J0xz81do8XmHeIFI",
+						CPF:        "761.647.810-78",
+						Secret:     "J0@0doR10",
+						Balance:    100,
+					}
+					return account, nil
+				},
+			},
+			input:   "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJVc2VySUQiOiJjMDM2NDc1Zi1iN2EwLTRmMzQtOGYxZi1jNDM1MTVkMzE3MjQifQ.Vzl8gI6gYbDMTDPhq878f_Wq_b8J0xz81do8XmHeIFI",
+			want:    1,
+			wantErr: false,
+		},
+		{
+			name: "dado id incorreto, retorna valor negativo e erro",
```

Comment:
Neste teste, fiquei confusa em onde esta o ID incorreto, ja que ele é igual ao teste acima.
Talvez o que voce esta querendo é testar o que acontece quando o repositorio retorna um erro. Se for o caso, seria só adequar o nome do teste.

Reply:
Corrigido, esqueci de mudar mesmo kkk